README.md

```diff
@@ -9,7 +9,7 @@ WorldEdit is Minecraft mod that turns Minecraft into an in-game map editor (sort
 * Making fjords and other natural wonders with weird names
 * Also: fix grief, broken water, missing snow, raging fires, and other :poop:
 
-WorldEdit is open source!, so if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
+WorldEdit is open source! So, if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
```

Comment:
```suggestion
**WorldEdit is open source**, so if you've just learned how to make a button blink, I probably recommend picking up a book on CS algorithms. The project is licensed using the GNU Lesser General Public License license, v3.
```

Reply:
This removes the exclamation! It totally undermines the emotional impact of the sentence! I think it's way more important to preserve the exclamation even if it changes the semantics!

Reply:
I think I'm going to request a CVE because that's how severe this is. It threatens my grammatical security.

Reply:
I agree with wiz's suggestion, it reads better.